packages/react-component-library/src/components/AutocompleteE/AutocompleteE.tsx

```diff
@@ -16,15 +16,53 @@ import { useAutocomplete } from './hooks/useAutocomplete'
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface AutocompleteEProps extends SelectBaseProps {}
 
+function onBlur(resetHighlightedIndex: () => void) {
+  return () => {
+    resetHighlightedIndex()
+  }
+}
+
+function onFocus(isOpen: boolean, openMenu: () => void) {
+  return () => {
+    if (!isOpen) {
+      openMenu()
+    }
+  }
+}
+
+function onKeyDown(
```

Comment:
Is it worth adding return types for these?

I try to explicitly add them but I get that the compiler can infer return types in some instances so maybe it's noise?

Reply:
It doesn't hurt 👍